packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -100,6 +106,42 @@ export const Canvas: React.FunctionComponent<CanvasProps> = ({
     [onMouseUp, hideDropSelector],
   );
 
+  const handleKeyDown = React.useCallback(
+    event => {
+      switch (
+        event.code // Begin switch 1
+      ) {
+        case 'Period': // Moves to the next element
+        case 'Comma': // Moves to the previous element
+          // if (event.ctrlKey) { // Begin if 1
```

Comment:
With this line commented out, you can navigate using just comma and period. That does not match the PR description.
Also what is the intentional hotkey on Mac? Usually whatever is *Control* on Windows we do *Command* on Mac. But *Command,* is already used on Mac.
I am ok with using *Control* regardless the platform.

Reply:
Sorry, the line should not be commented out. Control + comma and Control + dot should be used for navigation regardless of the platform, as mentioned in the PR description.